syncstorage-common/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
Do we want this file here? The setting crates PR removed the syncstorage-common crate

Reply:
Thanks, good catch! I thought this was deleted, merge fail.